pkg/api/audit.go

```diff
@@ -36,7 +36,7 @@ func GetAllAudit(c *gin.Context) {
 
 	_ = utils.PreparePaginateResponse(c, query, &models.AuditResponse{})
 
-	if err := query.Find(&audit).Error; err != nil {
+	if err := query.Order("id desc").Find(&audit).Error; err != nil {
```

Comment:
Please order by the timestamp as id might not always be in increasing order (or be integer in first place).

Reply:
Yepp